Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserBus.h

```diff
@@ -355,7 +355,10 @@ namespace AzToolsFramework
             virtual void ClearPreview(){};
 
             //! Preview the selected entry in the scene settings window, returns true if successful 
-            virtual bool PreviewSceneSettings([[maybe_unused]]const AzToolsFramework::AssetBrowser::AssetBrowserEntry* selectedEntry){ return false; };
+            virtual void PreviewSceneSettings([[maybe_unused]]const AzToolsFramework::AssetBrowser::AssetBrowserEntry* selectedEntry){};
+
+            //! Check if the source asset can be opened in the scene settings
+            virtual bool HandleSource([[maybe_unused]]const AzToolsFramework::AssetBrowser::AssetBrowserEntry* selectedEntry){ return false; };
```

Comment:
Can make this EBus function const
```suggestion
            virtual bool HandleSource([[maybe_unused]]const AzToolsFramework::AssetBrowser::AssetBrowserEntry* selectedEntry) const { return false; };
```

Reply:
Done!